src/Interfaces/DynamoTestStation.ts

```diff
@@ -15,4 +17,37 @@ interface DynamoTestStation {
   testStationType: string;
 }
 
-export { DynamoTestStation };
+const TestStationType = new Map<number, string>([
```

Comment:
Do these belong in an enums file or similar rather than interfaces?

Reply:
I'll move the enums and function in this file to a separate file